### DISCUSSION TITLE
Awesome work 🔥

### DISCUSSION THREAD

**Discussion by jjdenhertog** [oskvr37/tiddl#50] [General]
Hello!

Just a quick shout-out. Great to see that someone has continued building the new tidal-dl. I just implemented your app in [my repo](https://github.com/jjdenhertog/spotify-to-plex-tidal-downloader). Keep up the awesome work 💪 

Cheers,
Jaap

**Comment by oskvr37** [comment]
Hi, thanks very much, great to hear this! Im glad that you found my project useful 💪

**Comment by AHAULSS** [comment]
Hi @oskvr37, 
Thank you so much for building and sharing Tiddl! I was able to get it running even with very little command prompt experience. I also really appreciate you keeping it up to date and fixing the metadata issues — it makes the tool even more reliable and easy to use. Keep up the great work!
